### PR TITLE
DOCSP-29175 Add how to rename collection 

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
## DESCRIPTION
Rename a collection in mappings pane when creating mappings rules.

## STAGING
https://preview-mongodbjocelynmendez1.gatsbyjs.io/docs-relational-migrator/DOCSP-29175/mapping-rules/mapping-rule-options/rename-collection/

## JIRA
https://jira.mongodb.org/browse/DOCSP-29175

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6633a515b67dbf804a4e8295

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by ltran-mdb2 — APPROVED** [review]
LGTM! Thanks @jocelyn-mendez1 !

File: source/mapping-rules/new-rules-to-mongodb.txt [hunk 1]
```diff
@@ -54,6 +54,11 @@ Steps
    mapping rule, you can:
 
    - Change the collection name. 
+
```
Comment (alwaysbemark) [inline source/mapping-rules/new-rules-to-mongodb.txt:57]: I may be wrong about what this is referencing but the collection rename field is accessible "outside" of the configuration of any specific mapping 
image

Renaming the collection will only be possible if starting from the relational menu and adding a "New Document" mapping or clicking one of the MongoDB collections but not going into the "Add" menu.

Comment (tomhollander) [reply source/mapping-rules/new-rules-to-mongodb.txt:57]: Agree. I actually think this should go in a different topic, alongside where we put the time series / collection type docs.

Comment (jocelyn-mendez1) [reply source/mapping-rules/new-rules-to-mongodb.txt:57]: Thank you, I have added a small task page in that section!

**Review by tomhollander — APPROVED** [review]
Nice!

**Review by alwaysbemark — APPROVED** [review]

**Pull request merged**